pgoapi/pgoapi.py

```diff
@@ -142,9 +140,17 @@ def login(self, provider, username, password):
 
         self.log.debug('Auth provider: %s', provider)
 
-        if not self._auth_provider.login(username, password):
-            self.log.info('Login process failed')
-            return False
+        if auth_token is None:
+            if not self._auth_provider.login(username, password):
+                self.log.info('Login process failed')
+                return False
+            with open("token.txt", "w") as f:
```

Comment:
need to find a proper place to path this, man.... just this relative path is not good